pkg/bucket/client.go

```diff
@@ -58,7 +58,14 @@ func getCredentialFromCloudStorageSecret(a client.Client, cloudStorage v1alpha1.
 			return "", err
 		}
 
-		if common.CCOWorkflow() {
+		if stsSecret, err := stsflow.STSStandardizedFlow(); err != nil && stsSecret != "" {
+			err := a.Get(context.TODO(), types.NamespacedName{
+				Name:      stsSecret,
+				Namespace: cloudStorage.Namespace,
+			}, secret)
+			if err != nil {
+				return "", err
+			}
```

Comment:
STSStandardizedFlow could return secret data.. instead of secret name, eliminating 1 get call.